### ISSUE TITLE
Searching on etexts Part Two:

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#814]
Searching on etexts Part Two:
Search for a term, open an etext listed in the search results, and then have **a second search function to make additional and varied searches within that one etext**

_Originally posted by @JannTibetan in https://github.com/buda-base/public-digital-library/issues/803#issuecomment-1473643304_

**Comment by eroux** [comment]
so, basically a search function in the etext viewer right?

**Comment by JannTibetan** [comment]
Yes. Maybe one with a menu bar offering a couple of search tweaks

**Comment by JannTibetan** [comment]
Essentially, this intra-etext search bar would offer the functionality that isn't feasible on the homepage search bar (because that one is meant to search across the entire etext corpus)

**Comment by JannTibetan** [comment]
<img width="1438" alt="Screenshot 2023-03-17 at 7 13 45 AM" src="https://user-images.githubusercontent.com/44705185/225889862-3598d63b-e94d-4d81-99a7-dd54d3f23e2e.png">

**Comment by JannTibetan** [comment]
> just to be sure I get it, are we talking about this: https://library-dev.bdrc.io/show/bdr:IE0OPIBC4CB9AD
> 
> ## ![image](https://user-images.githubusercontent.com/33383857/225887880-69510aa4-3699-42db-817f-43e30651da02.png)
> leading to this: https://library-dev.bdrc.io/search?q=%22thog%20ma%27i%20%27od%20gsal%22&lg=bo-x-ewts&r=bdr:IE0OPI78ACB8B3&t=Etext&s=volume%20number
> 
> ![image](https://user-images.githubusercontent.com/33383857/225888176-c1875515-36eb-42be-963d-ca5b7403ac25.png)

This is a fantastic but under-used feature. @evanyerburgh can bring attention to it through the user Guide.

**Comment by eroux** [comment]
yes, search results scoped to a specific etext (or even etext segment) isn't something we provide yet, search is only at the level of the collection right now. I think it's a very good idea!
